src/components/common/Toaster/Toaster.tsx

```diff
@@ -1,16 +1,34 @@
 import { FC } from 'react';
 import { useStore } from 'store';
+import { ReactComponent as IconClose } from 'assets/icons/X.svg';
 
 export const Toaster: FC = () => {
   const { toaster } = useStore();
   return (
-    <ul className="fixed bottom-16 right-16 z-10">
+    <ul
+      id="toaster"
+      className="fixed bottom-16 right-16 z-10 flex min-w-[250px] flex-col gap-8"
+    >
       {toaster.toasts.map((toast) => (
         <li
+          id={toast.id}
           key={toast.id}
-          className="animate-slideUp bg-background-800 my-5 rounded-full border border-white/60 px-16 py-8"
+          className="from-background-900 to-primary-dark text-14 rounded-6 border border-white/10 bg-gradient-to-r"
         >
-          <output>{toast.content}</output>
+          {typeof toast.content === 'string' ? (
+            <output className="flex">
```

Comment:
to continue from next comment, maybe you'd want to create a ToastPresenter that generates toasts by type and populates with params passed to it. The toaster will be passed an instance of the presenter and ask it for components. The toast id logic for closing the toasts should only exist in one place - as a criteria for good design